redux-monitor/src/main/java/me/tatarka/redux/monitor/MonitorMiddleware.java

```diff
@@ -154,7 +155,7 @@ public void stop() {
     }
 
     @Override
-    public R dispatch(Next<A, R> next, A action) {
+    public R dispatch(Dispatcher<A, R> dispatcher, Next<A, R> next, A action) {
```

Comment:
By providing it here, you won't be able to dispatch any actions that aren't in response to another action, is the a reasonable limitation?

Reply:
For my use case, is not a problem, I would considerer this an edge. If someone need access to the dispatcher before they can pass it the constructor as we do today.